issue_template.md

```diff
@@ -16,7 +16,7 @@ kubectl version
 ```
 
 **Is Istio Auth enabled or not ?**
-Did you install the very mature istio.yaml, istio-auth.yaml....  or the immature Helm chart directly?
+Did you install the stable istio.yaml, istio-auth.yaml....  or the Helm chart?
 
```

Comment:
maybe ask their helm version for helm?

Reply:
Our docs specify a hard minimum on Helm and Helm is generally really stable as a tooling system at this point.  I think what would be more interesting data is if they used specific --set flags, but that opens pandora's box, especially in the official triage page.  Easier to ask that as a followup once triaged I think.

Cheers
-steve

Reply:
maybe ask for helm their full command line and output ? (if it's not too verbose)

anyway you can merge as is too, it's a good incremental improvment